airbyte-worker-models/build.gradle

```diff
@@ -0,0 +1,24 @@
+import org.jsonschema2pojo.SourceType
+
+plugins {
+    id "java-library"
+    id 'com.github.eirnym.js2p' version '1.0'
+}
+dependencies {
```

Comment:
Nit:  you can remove this if there are no additional dependencies required.  That being said, if the generated code has dependencies on third party libraries, we should declare them here (even if they are added in the base build.gradle) so it is documented.

Reply:
Fasterxml is added to the main build.gradle, I have removed the block